src/tooltip.js

```diff
@@ -76,7 +76,7 @@ class Tooltip {
         if (x != null && y != null)
             this.setPosition(x, y);
         if (!this.isOpen) {
-            this.getElement().style.display = "block";
+            this.getElement().style.display = "";
```

Comment:
is this required? this impacts all tooltips right?

Reply:
I reverted it. My original thought was to let it be defined by the CSS class, but this feels safer.